### DISCUSSION TITLE
JV-XP editor

### DISCUSSION THREAD

**Discussion by risktoparkrageram** [RomanKubiak/ctrlr#681] [Q&A]

**Comment by gearmo3ds** [comment]
I'v got me a JV 2080 and am looking for a CTRLR panel file, does it not exist?
I did not see it in the repository

**Comment by damiensellier** [comment]
**JV-XP Editor
Standalone, VST and AU Editor for Roland JV1080, JV2080, JV1010, XP30, XP50, XP60 and XP80 series synthesizers
 by dasfaker**

![JV-XP-Editor_1-768x534](https://github.com/user-attachments/assets/805176cf-0756-4bdd-bc87-dfa1d04dc155)


https://web.archive.org/web/20240425103014/https://ctrlr.org/jv-xp-editor-for-roland-jv1080-jv2080-xp30-xp50-and-xp80-series-synthesizers/

Mac :
https://drive.google.com/file/d/1uJKHjHb0fYg-P24Ok919wSY3NgmhPBtH/view
PC
https://drive.google.com/file/d/1oehyXHriUBzL3DP-T5lsg_DJ4T9ed3AY/view


**JV PatchEd.
Patch Editor for Roland JV and XP Synths (Roland JV-1010, JV-1080, JV-2080, XP-30, XP-50, XP-60, XP-80.)
by lecleto**

![jv-patched-jv-xp-3-768x420](https://github.com/user-attachments/assets/be1cb3a4-ac44-40b9-b12a-92c3b33dfa4a)

https://web.archive.org/web/20240714043216/https://ctrlr.org/jv-patched-roland-jv-xp-patch-editor/

https://sourceforge.net/projects/jv-patched-jv-xp/

**Reply by gearmo3ds** [reply]
Thank you so much (tysm) :)

**Reply by gybe808-dotcom** [reply]
Would love to have the JVXP midiforge Panel, sent a Google Drive access request!

**Comment by dobo365** [comment]
Impressive "web archive"!
Thx for the tip Damien!

**Reply by damiensellier** [reply]
I have the link to the panels' page in my shortcuts :)

https://web.archive.org/web/20240708171052/https://ctrlr.org/panels/
